fastlane/Pluginfile

```diff
@@ -6,7 +6,9 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 3.2.0'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.12.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', ref: '24db1b638b964ded7ea8a908a765ed8c60728722'
+#gem 'fastlane-plugin-wpmreleasetoolkit', path: '../../../wordpress-mobile/release-toolkit'
```

Comment:
🎗️ This is obvious that you plan to reverted this later in other incremental PRs, but just leaving a comment to highlight it so we don't forget later :)